src/main/java/bio/terra/service/dataset/DatasetRequestValidator.java

```diff
@@ -172,6 +172,8 @@ private void validateTable(TableModel table, Errors errors, SchemaValidationCont
         List<String> primaryKeyList = table.getPrimaryKey();
 
         if (tableName != null && columns != null) {
+            columns.forEach((column) -> validateDatatype(column, errors));
```

Comment:
minor nit, `()` are unnecessary if the anonymous function takes a single argument
```suggestion
            columns.forEach(column -> validateDatatype(column, errors));
```